templates/docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: "3.7"
+version: "3.2"
```

Comment:
Why do you decrease the version of dicker compose?

Reply:
MacOS
```
$ docker-compose version
docker-compose version 1.17.1, build 6d101fb
docker-py version: 2.5.1
CPython version: 2.7.12
OpenSSL version: OpenSSL 1.0.2j  26 Sep 2016
```
u can check here [Compose file versions and upgrading
](https://docs.docker.com/compose/compose-file/compose-versioning/)

Reply:
Version 1.17 was introduced more than 2 years ago. It supports up to v3.4 of the `docker-compose.yml`.

Could you please update Docker and Docker compose to the latest versions and check if it helps?

Reply:
Did you have a chance to check the build with updated Docker?